internetarchive/cli/ia.py

```diff
@@ -61,9 +61,9 @@
 import os
 import sys
 
-from docopt import docopt, printable_usage  # type: ignore
+from docopt import docopt, printable_usage
```

Comment:
This passes in pre-commit and GitHub Actions?  On older Py3 and newer Py3?

Reply:
I added printable_usage to typeshed:
https://github.com/python/typeshed/pull/7913

and it has just published:
https://pypi.org/project/types-docopt/0.6.10/